zenoh-ext/examples/z_query_sub.rs

```diff
@@ -55,24 +52,14 @@ async fn main() {
             .unwrap()
     };
 
-    println!("Enter 'q' to quit...");
-    let mut stdin = async_std::io::stdin();
-    let mut input = [0_u8];
+    println!("Press CTRL-C to quit...");
     loop {
         select!(
```

Comment:
Select is no longer necessary, just have a `while let Ok(sample) = subscriber.recv().await { .. }`.